src/app/core/signing.service.ts

```diff
@@ -78,17 +78,21 @@ export class SigningService {
     const start = textbox.selectionStart;
     const end = textbox.selectionEnd;
     const text = textbox.value;
+    const scroll_pos: number = textbox.scrollTop;
 
     // If the end character was not provided, we assume it will be the same as the start character
     if (typeof endChars === 'undefined' || !endChars) {
       endChars = formatChars;
     }
+    textbox.value = text.slice(0, start) + formatChars + text.slice(start, end) + endChars +
+      text.slice(end);
     this.signingBoxText = text.slice(0, start) + formatChars + text.slice(start, end) + endChars +
       text.slice(end);
```

Comment:
Can one of these be removed or can the variable assignment be merged into one command?
```suggestion
    textbox.value = this.signingBoxText = text.slice(0, start) + formatChars + text.slice(start, end) + endChars +
      text.slice(end);
```